op-batcher/flags/flags.go

```diff
@@ -82,6 +88,12 @@ var (
 		Value:   0,
 		EnvVars: prefixEnvVars("BATCH_TYPE"),
 	}
+	BatchSubmissionPolicy = &cli.StringFlag{
+		Name:    "batch-submission-policy",
+		Usage:   "The policy to use for submitting batches to the L1, e.g. blobs or calldata.",
+		Value:   CalldataPolicy,
+		EnvVars: prefixEnvVars("BATCH_SUBMISSION_POLICY"),
+	}
```

Comment:
Would you mind changing the flag like
```suggestion
	DataAvailabilityType = &cli.StringFlag{
		Name:    "da.type",
		Usage:   "The data availability type to use for submitting batches, e.g. blobs or calldata.",
		Value:   CalldataType,
		EnvVars: prefixEnvVars("DA_TYPE"),
	}
```
? 
Then also need to adapt a couple of error strings and variable names.

This might sound more generic for when we add other data availability layers in the future.

Reply:
done!